docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -34,6 +30,9 @@
 |spring.cloud.loadbalancer.health-check.initial-delay | 0 | Initial delay value for the HealthCheck scheduler.
 |spring.cloud.loadbalancer.health-check.interval | 25s | Interval for rerunning the HealthCheck scheduler.
 |spring.cloud.loadbalancer.health-check.path |  | 
+|spring.cloud.loadbalancer.health-check.refetch-instances | false | Indicates whether the instances should be refetched by the <code>HealthCheckServiceInstanceListSupplier</code>. This can be used if the instances can be updated and the underlying delegate does not provide an ongoing flux.
```

Comment:
Add backticks (for code) around "false". Change the last sentence to "You can use this setting if the instances can be updated and the underlying delegate does not provide an ongoing flux."

Reply:
This file is automatically generated.